build_tools/ci/run_matmul_test.sh

```diff
@@ -690,15 +690,29 @@ run_matmul_test \
     --lhs_rhs_type "bf16" \
     --acc_type "f32" \
     --m "512"  --n "512" --k "512" \
-    --num_repeat_runs "0"
 
 run_matmul_test \
     --name_prefix "packPeel2304" \
     --pipeline "pack-peel" \
     --lhs_rhs_type "bf16" \
     --acc_type "f32" \
     --m "128"  --n "128" --k "2304" \
-    --num_repeat_runs "0"
+
+run_matmul_test \
+  --name_prefix "packPeel_t_i32" \
+  --pipeline "pack-peel" \
+  --lhs_rhs_type "i32" \
+  --acc_type "i32" \
+  --m "128" --n "256" --k "512" \
+  --do_transpose_rhs "1"
+
+run_matmul_test \
```

Comment:
Nice one, first complete matmul transpose example I think? @jsetoain

Reply:
This is adding matmul transpose example for pack-peel pipeline. There's already a test working for pad-pack pipeline https://github.com/nod-ai/iree-amd-aie/commit/16a2c08ca13f97ba903335adcbbd90091040ee4b#diff-dfa2b1719749b22586f350c17128c1ec6531c32e18bb17968b75fea850f44333R558

Reply:
That's great!